### ISSUE TITLE
Transferring opentoonz from one pc to another - not getting all animation objects and not opening.

### ISSUE THREAD

**Issue by PantherTeaLord** [opentoonz/opentoonz#3097]
I wanted to transfer a animation from my laptop to my new pc;
 so for my first attempt i downloaded opentoonz on my new pc then on the laptop i ONLY copied the project folder onto a hard drive and replaced the download one on the new pc with the laptop one. (which of course has all my animations in it)
This did work however some objects were missing like music and two levels. (The levels were there but all the frames were red and no drawings were present in them.) 
On top of that, i cannot "save all" the animation. I can still save the scene though. (I would type the specific error message it gave me but i cant actually open the programme atm cause i am in the middle of trying to fix it still.)

My second attempt i just deleted opentoonz on the new pc and copied the entire opentoonz folder from my laptop onto the new pc. For this it just didn't open and said it was missing "SOFTWARE".

Now i know that openstoonz creates a folder in the C drive and then has the other folder in whatever drive? I put the main folder also in the c drive so i'm not quite sure how it fully works.

So in short my problem is i don't know how to fully transfer my projects unharmed. Opentoonz saves thing weirdly so its difficult to move.

(Sorry if i didn't explain things very well xD As a side note though, the hard drive i used to transfer opentoonz over was the literal D drive in my pc. I connected the D drive to my laptop by a usb cable and a power thing that lets one slot the drive into it.
If that make sense.)

**Comment by artisteacher** [comment]
Save All might be failing because of the missing levels. When you open the scene with the missing levels on the original computer, are they still missing? 

If the levels are still there, they might just be linked to the project and not actually be saved within the project itself. I usually import any external audio or etc. instead of loading to prevent this issue if I move the project later on.

To move a copy of the linked levels into the project, do the following:
1. Open an unrelated scene that doesn't use those levels (or start a new, empty scene)
2. In the Browser room, find the scene with the missing levels
3. Right-click on the scene and select "collect assets" from the context menu

Afterwards, if you look in that project's extras folder, the levels should be located there.

On the new computer, you probably need to remove everything and try a clean install. As long as all of the animation assets/levels are located within the project folders, you should be able to move the entire project folder (or individual project folders) without issue. Do be careful not to rename the projects when moving them.

**Comment by DarrenTAnims** [comment]
Your instincts were right. Copying the project folder should just work. However, if you've got drawings outside of the project, these won't be copied. You can follow what artisteacher suggested, but you can also just right click on a red level, choose level settings and that'll tell you where OpenToonz expects the files to be. Then you can copy them to the same location on the computer. Alternately, you can open the scene file in a text editor to see where they should be.

I made a video about this a month ago (because someone else asked me a similar question), that you might find useful: https://youtu.be/TiOfgSZ3BQ8

**Comment by RodneyBaker** [comment]
@PantherTeaLord   This is a follow up to see if you've found an ideal workflow for transferring Opentoonz and/or project files from PC to PC etc..

Any updates?

**Comment by PantherTeaLord** [comment]
Ahh sorry! I'm new to this website and always forget to check it. 
For the animation/s i was trying to transfers i decided to leave because some were cancelled and others were scrapped ideas. 

I really appreciate the info thats been given to me though! I'm sorry for wasting your time, i will try these things though when i go to transfer again sometime in the future.
The suggestions both from Art and darren seem like good ideas and things i didn't try! The right clicking on a red level is something that is useful to know that i didn't know before with other problems too haha.

Any ways, sorry for leaving this open for so long! I will close it as of now as the question does have answers though not tested. 
Sorry again and thank you for the help.
